### PR TITLE
`platform_freebsd.py` dummy test file

### PR DESCRIPTION
**Pull request by bigtedde**
Creates a dummy file for ACL testing FreeBSD. Relates to issue #7745.

### CODE REVIEW

**Comment by bigtedde** [comment]
@ThomasWaldmann is this similar to what you had in mind for a 'dummy file'?

File: src/borg/testsuite/platform_freebsd.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+@pytest.mark.skip(reason="not yet implemented")
+def test_access_acl():
+    pass
+
+
+@pytest.mark.skip(reason="not yet implemented")
+def test_default_acl():
+    pass
```
Comment (ThomasWaldmann) [inline src/borg/testsuite/platform_freebsd.py:25]: additionally to access and default, freebsd also has `nfs4` acls.

other than that: yeah, that's what I had in mind. like this, it is more obvious that there is something missing and maybe it motivates some developer to add that.

Comment (bigtedde) [reply src/borg/testsuite/platform_freebsd.py:25]: Excellent, I have added that additional test and just committed the changes.

**Comment by ThomasWaldmann** [comment]
OK, looks good. This is finished, right?

**Comment by bigtedde** [comment]
> OK, looks good. This is finished, right?

All set. Marked ready for review 👍

File: src/borg/testsuite/platform_freebsd.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+""" Dummy file for now, will eventually contain FreeBSD ACL tests """
```
Comment (ThomasWaldmann) [inline src/borg/testsuite/platform_freebsd.py:1]: Oh, didn't see this initially:
- either `"""bla bla"""
- or:
```
"""
bla
bla
"""

Comment (ThomasWaldmann) [reply src/borg/testsuite/platform_freebsd.py:1]: and add a dot at the end.

Comment (ThomasWaldmann) [reply src/borg/testsuite/platform_freebsd.py:1]: other than that, it's ready for merging.

Comment (bigtedde) [reply src/borg/testsuite/platform_freebsd.py]: All set 👍

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**